packages/airnode-node/src/handlers/process-transactions.test.ts

```diff
@@ -3,9 +3,9 @@ import * as evmHandler from '../evm/handlers/process-transactions';
 import * as fixtures from '../../test/fixtures';
 
 describe('processTransactions', () => {
-  test.each(['legacy', 'eip1559'] as const)('processes EVM providers', (txType) => {
+  test.each(['legacy', 'eip1559'] as const)('processes EVM providers - txType: %s', async (txType) => {
```

Comment:
I see we will have a few conflicts :D
I've implemented similar change in https://github.com/api3dao/airnode/pull/765

Reply:
![git merge](https://media.giphy.com/media/cFkiFMDg3iFoI/giphy.gif)